### DISCUSSION TITLE
The parameters in "ACGrGrPoly" kernel

### DISCUSSION THREAD

**Discussion by wenpeng231024** [idaholab/moose#17488] [Q&A Modules: Phase field] [answered]
Dear MOOSE Group,

I am trying to use "ACGrGrPoly" kernel to describe multiple grains and bubbles. The parameters mu, gamma, and kappa are different for matrix/matrix and matrix/bubble. So my question is how to set these parameters for  "ACGrGrPoly" kernel in the MOOSE? I found maybe the "GBAnisotropy" can do this. But how to specify the order of parameters for matrix/matrix and matrix/bubble?

Thanks,

Peng

**Comment by SudiptaBiswas** [comment, marked as answer]
You could use 'ACGrGrMulti' which allows you to provide multiple gamma values and corresponding order parameters.

**Reply by wenpeng231024** [reply]
Many thanks for your response. I found "ACGrGrMulti" is used for a multiple Grand Potential model. Does it also work for the KKS model? Because I use the KKS model.

Best,

Peng

**Reply by SudiptaBiswas** [reply]
MOOSE does not have the multi-phase 'KKS' model. For multi-phase, multi-order parameter cases, we use the grand potential model. However, think of `ACGrGrMulti` as an extension of `ACGrGrPoly` with multiple `gamma` inputs. So, if you were using `ACGrGrPoly` earlier, you can substitute it with `ACGrGrMulti`.

**Reply by wenpeng231024** [reply]
Ok. I see. I will try it. Thanks very much.

Peng
